charts/bindplane/templates/bindplane.yaml

```diff
@@ -244,6 +244,16 @@ spec:
             - name: BINDPLANE_GOOGLE_AUTH_CLIENT_ID
               value: {{ .Values.auth.google.clientid }}
             {{- end }}
+            {{- if eq .Values.auth.type "auth0" }}
+            - name: BINDPLANE_AUTH_TYPE
+              value: auth0
+            - name: BINDPLANE_AUTH0_CLIENT_ID
+              value: {{ .Values.extraEnv.auth.auth0.clientid }}
+            - name: BINDPLANE_AUTH0_DOMAIN
+              value: {{ .Values.extraEnv.auth.auth0.domain }}
+            - name: BINDPLANE_AUTH0_AUDIENCE
+              value: {{ .Values.extraEnv.auth.auth0.audience }}
+            {{- end }}
```

Comment:
You can do something like this:

```yaml
            {{- if len .Values.extraEnv }}
            {{- toYaml .Values.extraEnv | nindent 12 }}
            {{- end }}
```

12 may or may not be the right indent value. You can verify with `helm template charts/bindplane --values <some values file>`

Reply:
Thanks, 12 was correct
![image](https://github.com/observIQ/bindplane-op-helm/assets/1297166/e7a6e4fd-1408-4e5e-8d5a-a891360aa45f)